client/components/EditableText/index.js

```diff
@@ -57,8 +58,8 @@ export const EditableInput = ({
             setEditTitle(false);
             titleRef.current.style =
               'font-size:20px !important;font-weight:bold;color: #22c67c;';
-
-            dispatch(editTrip(tripDetails));
+            useEditTrips(tripDetails)
```

Comment:
pretty sure we should be destructuring the mutation func out of here and calling it ? or am I mistaken

Reply:
My bad. I'd fix that right away